versioned_docs/version-8.6/apis-tools/community-clients/cli-client/cli-get-started.md

```diff
@@ -4,6 +4,12 @@ title: Getting started with the CLI client
 sidebar_label: "Getting started with the CLI client"
 ---
 
+::: Heads up!
```

Comment:
shouldn't it be:
```suggestion
:::note Heads up!
```
?

Reply:
Yes, fingers moving faster than brain. Will fix.